tests/CMakeLists.txt

```diff
@@ -118,14 +118,11 @@ foreach(_category ${_categories})
 
   file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${_category})
 
+  set(_redirect > /dev/null)
   if(DEAL_II_MSVC)
-    set(_command ${CMAKE_COMMAND} -G${CMAKE_GENERATOR} ${_options} ${_category_dir})
-  else()
-    # Do not pass the generator with -G so that we use make instead of ninja
-    # for the test projects. This is because calling ninja several times in
-    # parallel for the same project will break the configuration.
-    set(_command ${CMAKE_COMMAND} ${_options} ${_category_dir} > /dev/null)
+    set (_redirect)
   endif()
+  set(_command ${CMAKE_COMMAND} -G${CMAKE_GENERATOR} ${_options} ${_category_dir} ${_redirect})
```

Comment:
```suggestion
  set(_command "${CMAKE_COMMAND} -G${CMAKE_GENERATOR} ${_options} ${_category_dir} ${_redirect}")
```

Reply:
Same here. Quoting the command will lead to the following command line option:
```
"/usr/bin/cmake -GNinja -DDEAL_II_DIR=/usr" -UDIFF_DIR -UNUMDIFF_DIR -UTEST_PICKUP_REGEX -UTEST_TIME_LIMIT -UTEST_MPI_RANK_LIMIT -UTEST_THREAD_LIMIT -UENABLE_PERFORMANCE_TESTS -UTESTING_ENVIRONMENT\ /home/tamiko/workspace/dealii/tests/base\ > /dev/null
```
Adding `VERBATIM` to the `add_custom_target` does not improve the situation but leads to funnier quoting:
```
"/usr/bin/cmake -GNinja -DDEAL_II_DIR=/usr" -UDIFF_DIR -UNUMDIFF_DIR -UTEST_PICKUP_REGEX -UTEST_TIME_LIMIT -UTEST_MPI_RANK_LIMIT -UTEST_THREAD_LIMIT -UENABLE_PERFORMANCE_TESTS "-UTESTING_ENVIRONMENT /home/tamiko/workspace/dealii/tests/base >" /dev/null
```